packages/store/src/store.ts

```diff
@@ -25,6 +35,7 @@ export class Store {
     @Inject(INITIAL_STATE_TOKEN)
     initialStateValue: any
   ) {
+    (this._selectableStateStream as ConnectableObservable<PlainObject>).connect();
```

Comment:
I think that there may be an issue with the explicit `connect()` where, if the subscriber count reaches 0, then the stream will disconnect and I think it won't reconnect. Potentially we would want to add a `refCount()` after the `publishReplay(1)` pipe in the observable declaration and get rid of the manual `connect()`.
We definitely need to have a test that checks a subscription to the select and then subscription (taking the subscriptions to zero) and then after a state change, subscribing to a select again and checking it gets the latest state as well as further updates.
This article goes into great depth to explain the differences:
https://itnext.io/the-magic-of-rxjs-sharing-operators-and-their-differences-3a03d699d255